payloads/v9/gamesdk.ts

```diff
@@ -0,0 +1,304 @@
+/**
+ * Types extracted from https://discord.com/developers/docs/game-sdk/sdk-starter-guide
+ */
+
+import type { Snowflake } from '../../globals';
+import type { LocalizationMap } from '../common';
+
+export interface APIAchievement {
+	/**
+	 * Unique id of the application
+	 */
+	application_id: Snowflake;
+	/**
+	 * Name of this achievement
+	 */
+	name: string;
+	/**
+	 * Localization dictionary for the `name` field
+	 */
+	name_localizations: LocalizationMap | null;
+	/**
+	 * The description of this achievement
+	 */
+	description: string;
+	/**
+	 * Localization dictionary for the `description` field
+	 */
+	description_localizations: keyof LocalizationMap | null;
```

Comment:
localization is different in v9 and v10

v9 docs: https://github.com/discord/discord-api-docs/blob/a4daa52a299019782bff305f1b7d1f756d09ace1/docs/game_sdk/Achievements.md#data-models

Reply:
this doesn't seem resolved

Reply:
i had changed the files but probably discarded the changes 💀, thanks for letting me know